### PR TITLE
GH-8 Adding contribution guide

### PR DESCRIPTION
**Pull request by kevinrodriguezGL**
- [x] Added contribution guide
- [x] Added contributors list

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# How to Contribute
+
+Winston is one of Gorilla Logic projects which is under active development and is also being used to provide information to all collaborators about general inquiries like available PTO, nearby wellness events and others. Hopefully this document makes the process for contributing clear and answers some questions that you may have.
```
Comment (greivinlopezgl) [inline CONTRIBUTING.md]: Update "nearby wellness events and others" with "upcoming wellness activities and others"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+The best way to get your bug fixed is to provide a reduced test case. Feel free to create a Github Issue with your bug description and steps to reproduce.
+
+## How to Get in Touch
+
+Contact any of the core contributors, you can find them on `CONTRIBUTORS.md`
+
+## Proposing a Change
+
+If you intend to change the bot, or make any non-trivial changes to the implementation, we recommend filing an issue. This lets us reach an agreement on your proposal before you put significant effort into it.
```
Comment (greivinlopezgl) [inline CONTRIBUTING.md:41]: "we recommend filing" -> "we recommend **filling**"

**Review by greivinlopezgl — APPROVED** [review]
Besides the two updates in the text I think this is outstanding.
Thanks for all your collaboration Kevin!

File: CONTRIBUTING.md (hunk 1 above)
Comment (kevinrodriguezGL) [reply CONTRIBUTING.md]: Done!

File: CONTRIBUTING.md (hunk 2 above)
Comment (kevinrodriguezGL) [reply CONTRIBUTING.md:41]: Filing as in 
> Filing a Police Report 

I think I'll leave this one as-is.

**Pull request merged**